### ISSUE TITLE
training Hyperparameter

### ISSUE THREAD

**Issue by Ivamcoder** [bearpaw/pytorch-pose#78]
Hi @bearpaw 
How have you chosen the batch size, learning rate changes? Is it like Original paper?
